include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf_dual_splitting.cpp

```diff
@@ -631,7 +644,13 @@ TimeIntBDFDualSplitting<dim, Number>::projection_step()
     if(this->print_solver_info() and not(this->is_test))
     {
       this->pcout << std::endl << "Explicit projection step:";
-      print_wall_time(this->pcout, timer.wall_time());
+      if(this->param.spatial_discretization == SpatialDiscretization::HDIV)
+      {
+        this->pcout << std::endl << "(mass system solved)";
+        print_solver_info_linear(this->pcout, n_iter_mass, timer.wall_time());
+      }
+      else
+        print_wall_time(this->pcout, timer.wall_time());
```

Comment:
Here, we should also use something like 

if(L2) - else if(HDIV) - else(AssertThrow(false))

Currently the code reads like we do not have to solve a mass matrix system in case of continuous finite elements, which is not the case and should therefore be avoided.